src/hooks/helpers/motionHookHelpers.ts

```diff
@@ -0,0 +1,51 @@
+import { BigNumber } from 'ethers';
+import { getStakeFromSlider } from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget';
```

Comment:
Yeah, I second that.

Even though this is super specific, I think that for "extreme" cases we can make exceptions and add an specific alias

Reply:
I will be moving these hooks into their component folders as over a few separate github comments with a few different people we seem to be converging on the idea that component-specific hooks should live with components and leave the top level hooks folder for hooks reused around the app. So that will solve the ugly import
#354